dune-project

```diff
@@ -60,14 +60,13 @@ possible and does not make any assumptions about IO.
   astring
   camlp-streams
   (ppx_expect (and (>= v0.15.0) :with-test))
-  (ocamlformat (and :with-test (= 0.24.1)))
+  (ocamlformat (and :with-test (= 0.26.1)))
```

Comment:
I moved this one to https://github.com/ocaml/ocaml-lsp/pull/1303

Reply:
I think it should be `0.26.2` (since `0.26.1` is not compatible with `5.2.0` (https://ocaml.org/p/ocamlformat/0.26.1)?

Reply:
Yes, but #1303 targeted master which is still OCaml < 5.1.
I will update this PR to actually perform the bump from 0.26.1 to 0.26.2.